### DISCUSSION TITLE
Right start parameters

### DISCUSSION THREAD

**Discussion by maxnadig** [neherlab/covid19_scenarios#306] [General] [answered]
I'm trying to simulate for CHE-AR and am not 100% certain about three params:

Initial suspected cases, default is set to 3 for March 1st, is this a preset according to AR or is it some default value. What exactly does it mean, can you give me an example?
Can I keep the date at march one or do I have to move it to the first initial detected case?
Regarding age based isolation, how does this affect the simultaion? Like a age specific mitigation, or are these people fully removed from the simulation?
Thank you for your help, best,
Max

**Comment by nnoll** [comment, marked as answer]
The default estimates are getting overhauled by direct fitting a few parameters to the case count data that is widely available now. 

For now you can think of age-based isolation as a reduction in social contacts for each age-group. We don't explicitly remove people from the simulation, we reduce selectively reduce their infectivity. Thus 60% isolation means that each person has 60% less contact with people. So the former, age-specific mitigation.

**Reply by maxnadig** [reply]
thx very much!
